workspace-manager-clienttests/src/main/java/scripts/testscripts/GetDataReferenceByName.java

```diff
@@ -0,0 +1,56 @@
+package scripts.testscripts;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.client.ApiClient;
+import bio.terra.workspace.client.ApiException;
+import bio.terra.workspace.model.CreateDataReferenceRequestBody;
+import bio.terra.workspace.model.CreateWorkspaceRequestBody;
+import bio.terra.workspace.model.CreatedWorkspace;
+import bio.terra.workspace.model.DataReferenceDescription;
+import bio.terra.workspace.model.ReferenceTypeEnum;
+import bio.terra.workspace.model.WorkspaceDescription;
+import java.util.List;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.utils.DataReferenceUtils;
+import scripts.utils.WorkspaceManagerServiceUtils;
+import scripts.utils.WorkspaceTestScriptBase;
+
+public class GetDataReferenceByName extends WorkspaceTestScriptBase {
```

Comment:
Hmm. I have something very similar at https://github.com/DataBiosphere/terra-workspace-manager/pull/182/files for PF-328

Reply:
Agreed. I think we only need one as long as it's in the integration test suite